src/app/core/params.ts

```diff
@@ -66,7 +59,7 @@ export default class Params {
      * @returns Returns parameter
      * @memberof Params
      */
-    getParameter(key) {
+    getParameter(key: string) {
```

Comment:
Get rid of this, figure out where this is being used, and replace it to use `getValue()`